qemu/tests/cfg/resize_qemu_img.cfg

```diff
@@ -6,6 +6,15 @@
     type = resize_qemu_img
     images = "test"
     image_name_test = images/test
+    variants:
+        - @default:
+            preallocation = off
```

Comment:
if we set preallocation=off here, we always set it explicitly in qemu-img command line, although this is the default settting, so do we need to move it under with_preallocation, then for shrink testing:
  shrink.default
  shrink.with_preallocation.off
for increase size testing, we also add a new case increase_size.with_preallocation.off

This could add more cases, that might not be good. But it can cover:
qemu-img resize xxx  (by default, preallocate=off)
qemu-img resize preallocate=off xxx (explicitly set preallocate=off)

Reply:
agree with you, I will update it. thanks.